tests/test_calendar_privacy.py

```diff
@@ -0,0 +1,63 @@
+from app.internal.calendar_privacy import can_show_calendar
+# after user system is merged:
+# from app.internal.security.dependancies import CurrentUser
+from app.routers.user import create_user
+
+
+USER_DATA = {
+    'username': 'new_test_username',
+    'email': 'new_test.email@gmail.com',
+    'password': 'new_test_password',
+    'language': 'english',
+    'language_id': 1
+}
+
+
+def test_can_show_calendar_public(session):
```

Comment:
Maybe we should prefer using the `user` fixture? :)

Reply:
I did not notice we had that! awesome thanks!